pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_guidance-on-using-xrefs.adoc

```diff
@@ -0,0 +1,55 @@
+[id="guidance-on-using-xrefs_{context}"]
+= Guidance on using xrefs
+
+[role="_abstract"]
+To ensure that internal cross-references, xrefs, work when you publish your content, use only inter-document xrefs. Although AsciiDoc supports different forms of xref, {ProductShortName} validates only inter-document xrefs. If validation fails, {ProductShortName} flags invalid xrefs.
+
+NOTE: {ProductShortName} only validates xrefs within a repository.
+
+The following are the syntaxes of inter-document xref:
+
+* `\xref:document-b.adoc#target-section[Target section]`
+* `\xref:../modules/document-c.adoc[Content C]`
```

Comment:
Who is responsible for ensuring that `../modules` directory will exist for the lifetime of the title? (ie is there a contract for url structure as delivered by pantheon?

Reply:
Interesting point, I thought that it did not matter where the file was. @xdavidson is there a requirement that the repos follow a specific structure for xrefs to work?  

For example, if the module `document-a` was in `topics/xyz/` and module `document-b`, was in `topics/abc/` would an xref in `document-b` - `xref:../xyz/document-a.adoc[Content A]` work?

Thanks Paul, I'll update the text based on the answer!

Reply:
@theashiot @pwright Using xrefs is not dependent on any particular directory structure or naming conventions.

Paul, if you have a file in a directory name `modules` and then at some point change that directory name from `modules` to something else, such as `smurfs`, then you will need to:
* reimport/update your documentation repo in Pv2
* change any xrefs to target `smurfs` instead of `modules`
These are not specific to xrefs.
Did I answer your question @pwright?

Reply:
@stoobie yes, I think so... my understanding of your answer is that Pv2 deploys the directory structure of your repo and therefore you can create xrefs that work in plain old asciidoc- Pv2 will allow those resolve correctly.

Reply:
@pwright Yes. Pv2 imports your Git repo as-is.